app/views/restaurants/index.html.erb

```diff
@@ -1,30 +1,12 @@
 <div class="row">
   <% @restaurants.each do |restaurant| %>
-    <section class="col-xs-3">
-      <%= card_image_div(restaurant, current_user) %>
+    <section class="col-md-4">
+      <div class="card image" style="background-image: url(<%= restaurant.image.url(:thumb).to_s %>);" onclick="vote(<%= restaurant.id %>, '<%= restaurant.name %>' );">
         <paper-ripple class="recenteringTouch" fit></paper-ripple>
```

Comment:
It just about eliminate the polymer no everything else. Please review it once again
Add a line note